config.json

```diff
@@ -1,10 +1,176 @@
 {
-  "track_id": "clojurescript",
-  "language": "ClojureScript",
-  "active": false,
-  "blurb": "",
-  "ignore_pattern": "[Ee]xample",
-  "solution_pattern": "[Ee]xample",
-  "test_pattern": "[Tt]est",
-  "exercises": null
+    "language": "ClojureScript",
+    "track_id": "clojurescript",
+    "active": true,
+    "blurb": "",
```

Comment:
Could you perhaps add a blurb? It is a short description of the language. See e.g. https://github.com/exercism/csharp/blob/master/config.json#L4